lib/services/ios-debug-service.ts

```diff
@@ -238,7 +239,7 @@ class IOSDebugService implements IDebugService {
                 tnsIosPackage = path.resolve(this.$options.frameworkPath);
             } else {
                 var platformData = this.$platformsData.getPlatformData(this.platform);
-                tnsIosPackage = this.$npmInstallationManager.install(platformData.frameworkPackageName).wait();
+                tnsIosPackage = this.$npmInstallationManager.getCachedPackagePath(platformData.frameworkPackageName, frameworkVersion);
```

Comment:
if the cache had been cleaned, will this add the dependency with its correct version to the cache?